packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/skema/SkemaRustProxy.java

```diff
@@ -39,4 +41,10 @@ public interface SkemaRustProxy {
 	@GET
 	@Path("/models/{modelId}/named_opos")
 	Response getModelNamedOpos(@PathParam("modelId") String modelId);
+
+
+	@PUT
```

Comment:
This is SKEMA's API, take it up with them 😆 

Also we need to send a body-content, which isn't normally associated with GET. (Since we are sending in mathML it can exceed URL-length so it needs to be a PUT or GET).

Reply:
In that case I was completely wrong, you are right, a PUT is the way to go.